### ISSUE TITLE
React Breadcrumb issue

### ISSUE THREAD

**Issue by christopherGdynia** [telekom/scale#737]
Hello there,

I am using React 17, React-Router 5.2 and scale beta 35, tried it before with beta 24.

When using the ScaleBreadcrumb, the anchor elements inside it, will be transformed to normal behaviour.

I used the Link elements inside the Breadcrumb, as well as normal anchor tags with onclick eventlisteners, but after clicking both of them, the page will fetch from the server causing a window reload.

Is this intentional or a bug in Stencil with React?

Thanks for your help. :)

**Comment by acstll** [comment]
Hey @christopherGdynia :) 

Ideally you should be able to use elements other than `a` tags. But I'm not really sure if this is a bug or the component architecture/design causing trouble.

Could you be so kind and post a snippet of your code?

**Comment by christopherGdynia** [comment]
Hey @acstll 

I've created a [Sandbox](https://codesandbox.io/s/optimistic-sea-f40vg?file=/src/index.tsx)

**Comment by acstll** [comment]
Thank you @christopherGdynia We're taking a look!

**Comment by acstll** [comment]
Hey @christopherGdynia, @maomaoZH and I had been trying to find a fix for this one and it's difficult without heavily refactoring the scale-breadcrumb component (which we don't want to do just yet).

But I think we found a solution that works with React and your router.

I created another sandbox to demo it: https://codesandbox.io/s/hopeful-mestorf-w9h0g?file=/src/App.tsx

The idea is to use the "onClick" technique, but attach the listener to the `scale-breadcrumb` element instead of to each one of the links, and use [`event.composedPath()`](https://javascript.info/shadow-dom-events#bubbling-event-composedpath) to access the actual `<a>` element triggering the event, which is "hidden" inside the shadow root so the router is not smart enough to see it.

Let me know what you think.
